docs/chain/fullnode.md

```diff
@@ -0,0 +1,140 @@
+---
+id: fullnode
+title: How to set up a Full Node
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+We will guide you through the process of setting up and connecting to a full node.
+In contrast to a collator, full nodes to not author blocks.
+They act as a backend for Websites, verify new blocks and validate extrinsics (e.g. coin transfers and other transactions) before they are gossiped to the collator nodes.
+
+## Setup
+
+There are currently two different runtimes (i.e., two different parachain environments) that a KILT full node can be part of:
+
+- peregrine: the public test network whose runtime is as close to the official chain as possible. It can be used to try stuff out before executing them on the live chain, which involves spending tokens that have real monetary value.
+- spiritnet: the official public network, which contains only stable features.
+
+:::info
+The remainder of the guide explaining how to run a full node is for the official spiritnet.
+Nevertheless, we recommend to try out the setup on our peregrine testnet first.
+Hence, at each step where it is applicable, we indicate what differs between the peregrine and spiritnet configuration for the full node to join either network.
+:::
+
+### WASM runtime execution
+
+A KILT full node should use the `--execution=wasm` parameter for both the relaychain and parachain collation.
+The alternative to WASM runtime execution is native runtime execution, which might be faster but can, in some cases, deviate from the WASM execution logic and result in a different state.
+When this happens the full node will crash and will stop synchronizing with the network.
+Since the WASM runtime logic is part of the blockchain state itself and hence represents the single source of truth, all nodes should execute the WASM version of the runtime logic.
+
+### Specify the right chain spec
+
+The `--chain` parameter decides which blockchain the KILT full node will join.
+This parameter must be specified for both the parachain and the relaychain, since both chains are, as a matter of fact, separate blockchains.
+The KILT parachain accepts an additional parameter to select the environment to use for the WASM runtime execution.
+This can either be `peregrine` or `spiritnet`.
+
+Hence, to start a full node for the spiritnet network, the parameter would be `--chain=spiritnet`, while for peregrine would be `--chain=peregrine`.
+
+### Where are all the files stored?
+
+The `--base-path` parameter specifies where all the persistent files must be stored.
+By default the session keys will also be stored in the _base path_, but we recommend to separate them from the other files.
+This makes sure that the keyfiles are not accidentally lost or published when the blockchain database is backed up or restored.
+
+<Tabs
+groupId="exec-strategy"
+defaultValue="Binary"
+values={[
+{label: 'Binary', value: 'Binary'},
+{label: 'Docker', value: 'Docker'},
+]}>
+
+<TabItem value="Binary">
```

Comment:
sorry to be so picky, but i would add a headline above the tabs. Otherwise it looks like the tabs belong to the section above.